sootup.core/src/main/java/sootup/core/graph/StmtGraph.java

```diff
@@ -145,7 +145,12 @@ public int degree(@Nonnull Stmt node) {
    */
   public abstract boolean hasEdgeConnecting(@Nonnull Stmt source, @Nonnull Stmt target);
 
-  /** returns a list of associated traps */
+  /**
+   * returns a list of associated traps
+   *
+   * <p>Note: you can use getTraps() but exceptionional flow information is now easier accessible on
+   * each Block
+   */
   @Deprecated
```

Comment:
Should we not remove the `@Deprecated` annotation if it is still fine in cases where performance is not critical and you just want a convenient way to access the traps?

Reply:
maybe rename it to show that its not just a getter anymore compared to Soot?

Reply:
Yes, could also work.